lib/SILGen/SILGenApply.cpp

```diff
@@ -4899,6 +4899,60 @@ RValue SILGenFunction::emitApplyAllocatingInitializer(SILLocation loc,
   return result;
 }
 
+/// Emit an application of the given method.
+RValue SILGenFunction::emitApplyMethod(SILLocation loc, ConcreteDeclRef declRef,
```

Comment:
Are there any other opportunities to use this in SILGen?

Reply:
I wouldn't want to speculate too much, but I think .appendLiteral in string interpolation will probably need to use this.